package.json

```diff
@@ -28,11 +28,13 @@
     "@types/rimraf": "^2.0.2",
     "@types/semver": "^6.2.0",
     "@types/stack-trace": "^0.0.29",
+    "@types/swagger-schema-official": "^2.0.20",
     "@types/yargs": "^11.1.1",
     "@zeit/next-css": "1.0.1",
     "@zeit/next-mdx": "^1.2.0",
     "@zeit/next-typescript": "^1.1.1",
     "algoliasearch": "^3.32.1",
+    "axios": "^0.19.0",
```

Comment:
I'm surprised that this addition didn't change the `yarn.lock`

Reply:
It's already in there, just adding it as an explicit dependency now.